apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/poll/PollResultTable.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.dao.state.poll;
+
+import com.apollocurrency.aplwallet.apl.core.db.DbKey;
+import com.apollocurrency.aplwallet.apl.core.db.LongKey;
+import com.apollocurrency.aplwallet.apl.core.db.LongKeyFactory;
+import com.apollocurrency.aplwallet.apl.core.db.derived.ValuesDbTable;
+import com.apollocurrency.aplwallet.apl.core.entity.state.poll.PollOptionResult;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+
+@Singleton
+@Slf4j
+public final class PollResultTable extends ValuesDbTable<PollOptionResult> {
```

Comment:
Also I don't see unit test for component. Is it missing?

Reply:
I don't see any value in adding DAO tests here, because SQL is quite simple and used everywhere. 
Instead, I added unit tests to services.

Reply:
Unit tests to services are mocked, that is not quite the same, imho.